src/integ_test_resources/common/scripts/device_config_builder.py

```diff
@@ -96,11 +98,11 @@ def ssm_client(self, aws_config: AWSConfig):
             aws_access_key_id=aws_config.accessKey,
             aws_secret_access_key=aws_config.secretKey,
             aws_session_token=aws_config.sessionToken,
-            region_name=aws_config.defaultRegion
+            region_name=aws_config.defaultRegion,
         )
-        return session.client('ssm')
+        return session.client("ssm")
 
-    def aws_config_from_environment(self) -> AWSConfig:
+    def aws_config_from_environment(self) -> AWSConfig:  # noqa: F821
```

Comment:
Odd, is this the undefined name error? It is defined if so...

Reply:
It is defined, and yet flake8 complains about it :(. Seems to be similar to this closed issue (https://github.com/PyCQA/pyflakes/issues/427) but that seems to have been released. I haven't spent a lot of time trying to troubleshoot the internals though.